cannot build on arch: mips64

### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/dockertest/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/dockertest/blob/master/CONTRIBUTING.md).
- [ ] This issue affects my [Ory Network](https://www.ory.sh/) project.
- [ ] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [ ] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Describe the bug

building on arch mips64le platform, and got the error:
```
✗GOARCH=mips64le go build -mod vendor -o bin/app app/main.go 
# github.com/ory/dockertest/v3/docker/pkg/system
vendor/github.com/ory/dockertest/v3/docker/pkg/system/stat_linux.go:11:9: cannot use s.Rdev (variable of type uint32) as type uint64 in struct literal
```

### Reproducing the bug

`GOARCH=mips64le go build -mod vendor -o bin/app app/main.go `

### Relevant log output

```shell
vendor/github.com/ory/dockertest/v3/docker/pkg/system/stat_linux.go:11:9: cannot use s.Rdev (variable of type uint32) as type uint64 in struct literal
```


### Relevant configuration

_No response_

### Version
go version go1.18.7 linux/amd64
github.com/ory/dockertest/v3 v3.9.1

### On which operating system are you observing this issue?

Linux

### In which environment are you deploying?

Binary

### Additional Context

_No response_

--- a/docker/pkg/system/stat_linux.go
+++ b/docker/pkg/system/stat_linux.go
@@ -11,7 +11,7 @@ func fromStatT(s *syscall.Stat_t) (*StatT, error) {
 		mode: s.Mode,
 		uid:  s.Uid,
 		gid:  s.Gid,
-		rdev: s.Rdev,
+		rdev: uint64(s.Rdev),
 		mtim: s.Mtim}, nil
 }
 

